concepts/classes/about.md

```diff
@@ -0,0 +1,260 @@
+# About
+
+JavaScript includes the capabilities for object-oriented programming ([OOP][wiki-oop]).
+In OOP, you want to create objects (_instances_) from "templates" (_classes_) so that they include certain data and functionality.
+The data properties are called _fields_ in the OOP context, the function properties are called _methods_.
+
+JavaScript did not have classes at all before they were added to the language specification in 2015.
+And even though a `class` keyword is available nowadays, JavaScript is still a _prototype-based_ language.
+
+To understand what that means and how JavaScript actually works, we will go back to the time when there were no classes.
+
+## Prototype Syntax
+
+### Constructor Function
+
+In JavaScript, the template (class) is facilitated by a regular function.
+When a function is supposed to be used as such a template, it is called a _constructor function_ and the convention is that the function name should start with a capital letter.
+Instances (objects) are derived from the template using the `new` keyword when invoking the constructor function.
+
+```javascript
+function Car() {
+  // ...
+}
+
+const myCar = new Car();
+const yourCar = new Car();
+```
+
+It is important to note that in JavaScript, the instances and the constructor function keep a relationship to each other even after the instances were created.
+Every instance object includes a hidden, internal property referred to as `[[prototype]]` in the language specification.
+It holds a reference to the value of the `prototype` key of the constructor function.
+Yes, you read that correctly, a JavaScript function can have key/value pairs because it is also an object behind the scenes.
+
+Since 2015, `[[prototype]]` can be accessed via `Object.getPrototypeOf()`.
+Before that, it was accessible via the key `__proto__` in many environments.
+
+Do not confuse the prototype of an object (`[[prototype]]`) with the `prototype` property of the constructor function.
+
+### Instance Fields
+
+Often, you want all the derived objects to include some fields and pass some initial values for those when the object is constructed.
+This is facilitated via the [`this` keyword][mdn-this].
```

Comment:
```suggestion
This can be facilitated via the [`this` keyword][mdn-this].
```

Because the more advanced and flexible way is using `defineProperty` / `defineProperties`.